common/assets/scss/application.scss

```diff
@@ -40,6 +40,7 @@ $govuk-font-family: "Inter", system-ui, sans-serif;
 @import "@ministryofjustice/frontend/moj/components/organisation-switcher/organisation-switcher";
 @import "@ministryofjustice/frontend/moj/components/primary-navigation/primary-navigation";
 @import "@ministryofjustice/frontend/moj/components/badge/badge";
+@import "@ministryofjustice/frontend/moj/components/sortable-table/_sortable-table";
```

Comment:
You don't need to include the underscore in this definition. The [underscore pattern](https://sass-lang.com/guide#topic-4) is used to define partials in sass and to tell it not to turn that file in a `.css` file when you run the sass command on a folder or set of files.

When including partials using `@import` you don't need to specifically add the underscore. It would be neater to keep the same pattern we have in this file and also if the underscore is ever removed from the Design System it wouldn't break these references.

Reply:
It may be that you're right but when I tried without `_` I got a compilation error. The compiler was trying to include the js instead (!!). I am not entirely sure why.